refinery/ui/source/js/analysis-monitor/directives/global-status.spec.js

```diff
@@ -17,7 +17,7 @@
     ) {
       $templateCache.put(
         $window.getStaticUrl('partials/analysis-monitor/partials/global-status.html'),
-        '<div id="global-status"></div>'
+        '<div id="global-status""></div>'
```

Comment:
typo?

Reply:
@mccalluc Yes, I fixed it but forgot to push. Please refresh.